packages/pangea-node-sdk/src/services/audit.ts

```diff
@@ -573,6 +585,14 @@ class AuditService extends BaseService {
           publishedRoots: this.publishedRoots,
           record: record,
         });
+        if (record.consistency_verification === "fail") {
+          fixConsistencyProof(Number(record.leaf_index)).then(() => {
```

Comment:
The rest of the overarching `async` function won't wait for this to complete (but I think you want it to?), because you're using `then()` instead of `await`. I think it would be better if you turned the `forEach()` above into a for-of loop, and then use `await` here, or maybe even better: put all of the `Promise`s into an array and then `Promise.all()` them.